### PR TITLE
Update migrating.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
Add mmetl tool to the Slack migration docs

### CODE REVIEW

File: source/administration/migrating.rst [hunk 1]
```diff
@@ -190,7 +189,14 @@ Migrating from Slack Using the Mattermost CLI
    
 .. note:: To run the CLI command, you must be in the directory that contains the Mattermost installation. On a default installation of Mattermost, the directory is ``/opt/mattermost/``. Also, if you followed our `installation process <../guides/administrator.html#installing-mattermost>`__, you must run the command as the user *mattermost*. The executable is in the ``bin`` subdirectory and is called ``mattermost``.
 
+Migrating from Slack Using the Mattermost mmetl Tool and Bulk Import
+`````````````````````````````````````````````````````````````````````
+.. note:: This method is the recommended way to import Slack's corporate export file.
 
+1. Use the `mmetl tool <https://github.com/mattermost/mmetl>`_ to transform Slack's corporate export file into a the jsonl format required by the bulk import tool.
```
Comment (justinegeffen) [inline source/administration/migrating.rst:196]:
```suggestion
1. Use the `mmetl tool <https://github.com/mattermost/mmetl>`_ to transform Slack's corporate export file into a the ``jsonl`` format required by the bulk import tool.
```

File: source/administration/migrating.rst [hunk 2]
```diff
@@ -190,7 +189,14 @@ Migrating from Slack Using the Mattermost CLI
...
+
+2. Bulk load the files using the `bulk loading documentation <https://docs.mattermost.com/deployment/bulk-loading.html#bulk-loading-data>`_.
```
Comment (justinegeffen) [inline source/administration/migrating.rst:198]:
```suggestion
2. Bulk load the files using the steps provided in the `bulk loading documentation <https://docs.mattermost.com/deployment/bulk-loading.html#bulk-loading-data>`_. 
```

File: source/administration/migrating.rst [hunk 3]
```diff
@@ -190,7 +189,14 @@ Migrating from Slack Using the Mattermost CLI
    
 .. note:: To run the CLI command, you must be in the directory that contains the Mattermost installation. On a default installation of Mattermost, the directory is ``/opt/mattermost/``. Also, if you followed our `installation process <../guides/administrator.html#installing-mattermost>`__, you must run the command as the user *mattermost*. The executable is in the ``bin`` subdirectory and is called ``mattermost``.
 
+Migrating from Slack Using the Mattermost mmetl Tool and Bulk Import
+`````````````````````````````````````````````````````````````````````
+.. note:: This method is the recommended way to import Slack's corporate export file.
 
+1. Use the `mmetl tool <https://github.com/mattermost/mmetl>`_ to transform Slack's corporate export file into the ``jsonl`` format required by the bulk import tool.
```
Comment (svelle) [inline source/administration/migrating.rst:196]:
```suggestion
1. Use the `slack-advanced-exporter <https://github.com/grundleborg/slack-advanced-exporter>`_ to download attachments and add user's email addresses to your Slack corporate export file.

2. Use the `mmetl tool <https://github.com/mattermost/mmetl>`_ to transform Slack's corporate export file into the ``jsonl`` format required by the bulk import tool.
```

Added another important step if people want to have the attachments in their exports as this is currently not an inbuilt functionality of mmetl.

**Comment by justinegeffen** [comment]
Thanks, @svelle. I committed your suggestion. Have added @grundleborg for final review as his repo is mentioned.

File: source/administration/migrating.rst (hunk 3 above)
Comment (justinegeffen) [reply source/administration/migrating.rst]: Agreed. Additional step has been added.

File: source/administration/migrating.rst [hunk 4]
```diff
@@ -170,7 +169,7 @@ The following limitations are present when importing from Slack:
 Migrating from Slack using the Mattermost Web App
 `````````````````````````````````````````````````
 
-.. note:: For larger imports, particularly those where you have used the `slack-advanced-exporter tool` to add Slack post attachments to the archive or the Corporate Export file, it is recommended to import the Slack data using the `CLI <https://docs.mattermost.com/administration/migrating.html#migrating-from-slack-using-the-mattermost-cli>`__.
+.. note:: For larger imports, particularly those where you have used the `slack-advanced-exporter tool` to add Slack post attachments to the archive or the Corporate Export file, it is recommended to import the Slack data using the `mmetl tool and bulk loading tool <https://docs.mattermost.com/administration/migrating.html#Migrating-from-slack-using-the-mattermost-mmetl-tool-and-bulk-import>`__.
```
Comment (amyblais) [inline source/administration/migrating.rst:172]: FYI the link here is not working

Comment (justinegeffen) [reply source/administration/migrating.rst:172]: Thanks for picking this up, Amy!

**Pull request merged**